core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20250415_add_submission_entity_table.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.30.xsd">
+    <changeSet logicalFilePath="dummy" author="bmorgan" id="create_submission_entity">
+        <!-- create the SUBMISSION_ENTITY table -->
+            <createTable tableName="SUBMISSION_ENTITY">
+                <column name="submission_id" type="BINARY(16)">
+                    <constraints nullable="false" />
+                </column>
+                <column name="entity_id" type="BIGINT UNSIGNED">
+                    <constraints nullable="false" />
+                </column>
+            </createTable>
+<!--            <addUniqueConstraint columnNames="submission_id, entity_id"-->
```

Comment:
It seems reasonable to have a unique constraint here, but it breaks all the tests; I'm not sure if this is just an artifact of how the test data is set up and/or the tests run, or something meaningful.  But it might not be necessary, as experimentally it seems like even if there is a duplicate row it won't result in the same entity returned multiple times in a submission.